src/qibo_cloud_backends/backends/qibo_client.py

```diff
@@ -0,0 +1,31 @@
+import qibo_client
+from qibo.backends import NumpyBackend
+from qibo.config import raise_error
+
+PROVIDERS_LIST = [
+    "TII",
+]
+
+
+class QiboClientBackend(NumpyBackend):
```

Comment:
The `MeasurementOutcomes` object created by the execution heavily relies on the backend to calculate `frequencies`, `probabilities` and related things. Therefore, at the moment it needs to rely on the `NumpyBackend`, but I agree that in general it could be better to have a more comprehensive `AbstractBackend`.

Reply:
https://github.com/qiboteam/qibo/issues/1222